backend/test/integration/pipeline_version_api_test.go

```diff
@@ -278,6 +278,37 @@ func (s *PipelineVersionApiTest) TestPipelineVersionAPI() {
 	var expectedWorkflow v1alpha1.Workflow
 	err = yaml.Unmarshal(expected, &expectedWorkflow)
 	assert.Equal(t, expectedWorkflow, *template)
+
+	/* ---------- Update pipeline default version ---------- */
+
+	pipelineVersionParams = uploadParams.NewUploadPipelineVersionParams()
+	pipelineID := "testId15242123"
+	pipelineVersionParams.SetPipelineid(&pipelineID)
+	argumentYAMLPipelineVersion, err = s.pipelineUploadClient.UploadPipelineVersion("../resources/arguments-parameters.yaml", pipelineVersionParams)
+	assert.Nil(t, err)
+	assert.Equal(t, "arguments-parameters.yaml", argumentYAMLPipelineVersion.Name)
+
+	argumentYAMLPipelineVersion, err = s.pipelineUploadClient.UploadPipelineVersion("../resources/arguments-parameters.yaml", pipelineVersionParams)
+	assert.Nil(t, err)
+	assert.Equal(t, "arguments-parameters.yaml", argumentYAMLPipelineVersion.Name)
+
+	versions, _, _, err := s.pipelineClient.ListPipelineVersions(&params.ListPipelineVersionsParams{ResourceKeyID: &pipelineID})
```

Comment:
Do you know how does this sort by default? Shall we specify a sorting so it can be stable?